src/ShapeType.php

```diff
@@ -4,35 +4,35 @@
 
 namespace PhpMyAdmin\ShapeFile;
 
-final class ShapeType
+enum ShapeType: int
 {
-    public const NULL = 0;
+    case NULL = 0;
 
-    public const POINT = 1;
+    case POINT = 1;
 
-    public const POLY_LINE = 3;
+    case POLY_LINE = 3;
```

Comment:
Enum case names uses the PascalCase format.

```suggestion
    case PolyLine = 3;
```